storm-core/src/clj/org/apache/storm/converter.clj

```diff
@@ -71,6 +72,8 @@
                                                           (:worker->resources assignment)))))
     thrift-assignment))
 
+;TODO: when translating this function, you should replace the map-val with a proper for loop HERE
+;TODO: when translating this function, you should replace the map-key with a proper for loop HERE
```

Comment:
Only need one copy of this TODO.  I assume this will be cleaned up with a follow on JIRA